RUMM-2897 create release archive (#47)
DataDog/dd-sdk-roku: README.md

ropm install datadogroku
```

### Setup manually

> If your project doesn't use `ROPM`, you can install the library manually by downloading the [Roku SDK][5] zip archive, 

Reviewer: ```suggestion
If your project does not use `ROPM`, install the library manually by downloading the [Roku SDK][5] zip archive, 
```
